etc/security/limits.d/memlock.conf

```diff
@@ -0,0 +1,2 @@
+* hard memlock 2147484
```

Comment:
I think instead we should use `realtime-privileges` package and add user into realtime group.

Reply:
I noticed that `realtime-privileges` sets the `nice`ness to `11`

https://gitlab.archlinux.org/archlinux/packaging/packages/realtime-privileges/-/blob/main/99-realtime-privileges.conf?ref_type=heads#L3